<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RSA Encryption &amp; Decryption</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="generateKeysButton">
      <property name="text">
       <string>Generate Keys</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="plaintextEdit">
      <property name="placeholderText">
       <string>Enter plaintext here...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="encryptButton">
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="ciphertextEdit">
      <property name="placeholderText">
       <string>Encrypted text will appear here...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="decryptButton">
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="recoveredTextEdit">
      <property name="placeholderText">
       <string>Decrypted text will appear here...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>